packages/cashier/src/stores/__tests__/general-store.spec.ts

```diff
@@ -2,14 +2,15 @@ import { configure } from 'mobx';
 import { waitFor } from '@testing-library/react';
 import { routes, ContentFlag } from '@deriv/shared';
 import GeneralStore from '../general-store';
-import type { TWebSocket, TRootStore } from 'Types';
+import type { TWebSocket } from '../../types';
+import { mockStore, TStores } from '@deriv/stores';
```

Comment:
```suggestion
import { mockStore } from '@deriv/stores';
```
@rostislav-deriv Use `TRootStore` instead 🙏🏻